chassis.go

```diff
@@ -133,13 +151,38 @@ func (odbi *ovndb) chassisGetImp(hostname string) ([]*Chassis, error) {
 	return listChassis, nil
 }
 
+
 func (odbi *ovndb) rowToChassis(uuid string) (*Chassis, error) {
+
+	cacheChassis, ok := odbi.cache[tableChassis][uuid]
+	if !ok {
+		return nil, fmt.Errorf("Chassis with uuid%s not found", uuid)
+	}
 	ch := &Chassis{
-		UUID:       uuid,
-		Name:       odbi.cache[tableChassis][uuid].Fields["name"].(string),
-		Hostname:   odbi.cache[tableChassis][uuid].Fields["hostname"].(string),
-		ExternalID: odbi.cache[tableChassis][uuid].Fields["external_ids"].(libovsdb.OvsMap).GoMap,
-		NbCfg:      odbi.cache[tableChassis][uuid].Fields["nb_cfg"].(int),
+		UUID:                uuid,
```

Comment:
Still Encaps field is not populated.

Reply:
ohk I thought you wanted to add encap read support which I added. Sure i'll add it here too just to populate the uuid!